### ISSUE TITLE
Add google-service.json file to the project

### ISSUE THREAD

**Issue by mertceyhan** [muratcanbur/droidhub#8]
Adding `google-service.json` file to the project provides a more efficient development environment. There is no problem adding it to the project according to this blog post 👉🏻 [Publish the “google-services.json” file of an android app project or not?](https://cyb3rko.medium.com/publish-the-google-services-json-file-of-an-android-app-project-or-not-1951fb555b26)

**Comment by cyb3rko** [comment]
@mertceyhan Exactly. As long as you use security rules it's save and also as intended.
